person.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'nameable'
+require './rental'
+
+class Person < Nameable
+  attr_accessor :name, :age, :rentals
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, parent_permission)
+    # Call the superclass constructor with no arguments
+    @id = rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
- [ ] Nice work setting up **Setter** and **Getter** for the instance variable `@rentals` :clap: However, there are not an instance variable named `@ranaals` in your `Person` class. Kindly add an instance variable named `@rantals` to your `initialize` method and initialize that variable to an empty array (`[ ]`)
```ruby
class Person < Nameable
  ...

  def initialize(age, name = 'Unknown', parent_permission: true)
    ...
    @rantals = []
  end
...
end
```

Reply:
rentals[] initialised